app/templates/buyers/index.html

```diff
@@ -5,16 +5,17 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
-    {
-        "link": "/",
-        "label": "Digital Marketplace"
-    }
+  {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Digital Marketplace",
+        "href": "/"
+      },
+      {
+        "text": current_user.name
```

Comment:
Could we have this as `Your account` to be consistent with breadcrumbs for later pages please?

Reply:
Changed.